open-liberty/content.md

```diff
@@ -0,0 +1,148 @@
+# Overview
+
+The images in this repository contain Open Liberty. For more information about Open Liberty, see the [Open Liberty Website](https://openliberty.io/) site.
+
+# Images
+
+There are multiple images available in this repository.
+
+The `kernel` image contains the Liberty kernel and can be used as the basis for custom built images that contain only the features required for a specific application. For example, the following Dockerfile starts with this image, copies in the `server.xml` that lists the features required by the application.
+
+```dockerfile
+FROM %%IMAGE%%:kernel
+COPY server.xml /config/
+```
+
+The `microProfile1` image contains the features required to implement Eclipse MicroProfile 1.2. The `webProfile7` image contains the features required for Java EE7 Web Profile compliance. The `javaee7` image adds the features required for Java EE7 Full Platform compliance. The `javaee7` image is also tagged with `latest`.
+
+There are also additional images for different JVM combinations. Currently there are tags for java8 only, but there are two variants one based on IBM Java and Ubuntu and the other based on the IBM small footprint Java which is based on alpine linux. The naming structure for the variants is tag-javaversion-vandor/variant. This leads to webProfile7-java8-ibmsfj as one. At this time the full list of images are:
+
+	kernel
+	kernel-java8-ibm
+	kernel-java8-ibmsfj
+	microProfile1
+	microProfile1-java8-ibm
+	microProfile1-java8-ibmsfj
+	webProfile7
+	webProfile7-java8-ibm
+	webProfile7-java8-ibmsfj
+	javaee7
+	javaee7-java8-ibm
+	javaee7-java8-ibmsfj
+
+# Usage
+
+The images are designed to support a number of different usage patterns. The following examples assume that [DefaultServletEngine.zip](https://github.com/WASdev/sample.servlet/releases/download/V1/DefaultServletEngine.zip) has been extracted to `/tmp`.
+
+1.	Each image contains a default server configuration that specifies the corresponding features and exposes ports 9080 and 9443 for HTTP and HTTPS respectively. A .WAR file can therefore be mounted in the `dropins` directory of this server and run. The following example starts a container in the background running a .WAR file from the host file system with the HTTP and HTTPS ports mapped to 80 and 443 respectively.
+
+	```console
+	$ docker run -d -p 80:9080 -p 443:9443 \
+	    -v /tmp/DefaultServletEngine/dropins/Sample1.war:/config/dropins/Sample1.war \
+	    %%IMAGE%%:webProfile7
+	```
```

Comment:
Again, this tag isn't in the current image proposal -- is this just a bad copy-paste?

Reply:
see above